sales_report_product_image/tests/test_sales_report_product_image.py

```diff
@@ -22,5 +22,5 @@ def test_sale_report_product(self):
                 'pricelist_id': self.env.ref('product.list0').id,
                 'print_image': 'True',
                 'image_sizes': 'image_medium',
-                'image_small': self.product.image_small,
+                'image_small': self.name.image_1920,
```

Comment:
any reason for 1920?

Reply:
@JayVora-SerpentCS In version 13.0  **hr**  module employee image field name is **image_1920**,so we took that field.

Reply:
This is not HR. 1920 is size taken dynamicallly. Better you check with Chiragbhai.

Reply:
@JayVora-SerpentCS I see , it's my mistake. i will correct it.

Reply:
Improved Code as per the suggestion.

Reply:
@YogendrasinhChavda web_tree_image ni kharekhar jarur 6?

Reply:
@JayVora-SerpentCS: Actually that OCA module ( web_tree_image ) is to show the images in List view, but now we managed by adding the Height for the image field in the tree view. so, that module is not needed.
We Removed that from dependency.